examples/template.html

```diff
@@ -124,6 +124,10 @@
             animate();
           });
 
+          $(document).ready(function(){
```

Comment:
Hmmm, should we get rid of all occurrences of `$(function())...` in favor of `document`? at least that way we would guarantee that we only display stuff once we truly are ready to display it.